ibis/expr/api.py

```diff
@@ -1292,20 +1295,38 @@ def _integer_to_interval(arg, unit='s'):
 
 abs = _unary_op('abs', ops.Abs)
 ceil = _unary_op('ceil', ops.Ceil)
+degrees = _unary_op('degrees', ops.Degrees)
+deg2rad = _unary_op('radians', ops.Radians)
 exp = _unary_op('exp', ops.Exp)
 floor = _unary_op('floor', ops.Floor)
 log2 = _unary_op('log2', ops.Log2)
 log10 = _unary_op('log10', ops.Log10)
 ln = _unary_op('ln', ops.Ln)
+radians = _unary_op('radians', ops.Radians)
+rad2deg = _unary_op('degrees', ops.Degrees)
```

Comment:
Due to `rad2deg` and `deg2rad` are merely aliases for `degrees` and `radians` You don't need to define them again, just add to `_numeric_value_methods`:

```python
_numeric_value_methods = dict(
    degrees=degrees,
    radians=radians,
    rad2deg=degrees,
    deg2rad=radians
)
```